packages/core/helper-plugin/package.json

```diff
@@ -69,7 +69,7 @@
     "@strapi/icons": "1.10.1",
     "@testing-library/react": "14.0.0",
     "@testing-library/user-event": "14.4.3",
-    "@types/react": "18.2.16",
+    "@types/react": "18.2.7",
```

Comment:
This is the version of the DS and if we don't use the same one, `children` causes some issues 🤔